### PR TITLE
release:v1.27.1

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

**Comment by jennijuju** [comment]
@rvagg - i recall seeing a message either with Vulcanize or blockscout that #11952 in this release requires a migration in the SQLite db but it is not captured in upgrade warning or any release note, and that makes me a bit uncomfortable. could we add a quick note on either a migration guide or if its a small migration - at least what to expect when node operators w/ eth api enabled  should expect when they start a node with v1.27.1 ?

**Review by BigLep — APPROVED** [review]
I haven't been close to the changes for this release to give a very good spot check, but this seems right from my scan.  I'll give an "approve" in case this helps unblock and the risk is low.

**Comment by jennijuju** [comment]
Thanks, @BigLep! Given that thisill be included in the next stable release, I will wait for at least another review from the maintainers bef PR marks what wore merging!

File: .github/workflows/release.yml
Comment (rvagg) [inline .github/workflows/release.yml:93]: got some merge conflicts here

**Comment by rvagg** [comment]
4 differences between this and `release/v1.27.1`

1. The commit conflicts in `.github/workflows/release.yml` that have to be dealt with
2. This has a change that exists on `releases` but isn't on `release/v1.27.1` where `chore: pin golanglint-ci to v1.58.2` appears at the end of the changelog entry for v1.11.1!! I don't think that's a problem for this PR but something to keep an eye on when we merge back in to master.
3. One newline removal in `Makefile` at line 383. Inconsequential but weird.
4. One newline addition in `scripts/version-check.sh`. Inconsequential but weird.

So, good to go with the first one addressed.

**Comment by jennijuju** [comment]
> 4 differences between this and `release/v1.27.1`
> 
> 1. The commit conflicts in `.github/workflows/release.yml` that have to be dealt with
> 2. This has a change that exists on `releases` but isn't on `release/v1.27.1` where `chore: pin golanglint-ci to v1.58.2` appears at the end of the changelog entry for v1.11.1!! I don't think that's a problem for this PR but something to keep an eye on when we merge back in to master.
> 3. One newline removal in `Makefile` at line 383. Inconsequential but weird.
> 4. One newline addition in `scripts/version-check.sh`. Inconsequential but weird.
> 
> So, good to go with the first one addressed.

Thanks! Will fix 1,3,4. And 2 weirded me out as well when I resolved the conflicts. So did a quick check on the code itself and seems it was in place. But notes on the comment on when we merge to master

**Comment by jennijuju** [comment]
(i feel like there should be some checks that catch that not-fully-resolved conflicts but.. yah

**Review by rvagg — APPROVED** [review]
no diff to `release/v1.27.1`

**Pull request merged**
